tgtg_scanner/notifiers/smtp.py

```diff
@@ -44,6 +46,21 @@ def __init__(self, config: Config, reservations: Reservations, favorites: Favori
                 self._connect()
             except Exception as exc:
                 raise SMTPConfigurationError(exc) from exc
+            if config.smtp.recipients_per_item is not None:
+                item_recipients = None
+                try:
+                    item_recipients = json.loads(config.smtp.recipients_per_item)
+                    if not isinstance(item_recipients, dict) or any(
+                        not isinstance(value, (list, str)) for value in item_recipients.values()
+                    ):
+                        raise SMTPConfigurationError()
+                # Catches both json.decoder.JSONDecodeError at json.loads()
+                # and SMTPConfigurationError during key-value validation of item-recipients pairs inside
+                except Exception:
+                    raise SMTPConfigurationError("Recipients per Item is not a valid dictionary")
+                self.item_recipients = item_recipients
+            else:
+                self.item_recipients = {}
```

Comment:
Maybe move this part to the top before line 37 as the default value.

```suggestion
                self.item_recipients: dict[str, list[str]] = {}
```